CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 
 * Disabled a sleep in debug mode that was impairing external tests.
   PR [#2651](https://github.com/realm/realm-core/pull/2651).
+* Fix an assert that prevented `Group::commit()` from discarding history from a
+  Realm file opened in nonshared mode (via `Group::open()`, as opposed to
+  `SharedGroup::open()`).
```

Comment:
I guess this must have been branched from `master` exactly while I was doing the release and there was no `NEXT RELEASE` section?